apps/elewa-group-website/src/app/app.module.ts

```diff
@@ -6,12 +6,14 @@ import { ScullyLibModule } from '@scullyio/ng-lib';
 import { AppComponent } from './app.component';
 
 import { HomePageModule } from '@elewa-group/pages/elewa/home'
+import {CardsModule} from 'libs/features/components/cards/src/lib/cards/cards.module'
```

Comment:
Use domain imports i.e 
import { CardsModule } from '@elewa-group/features/components/cards'

Reply:
Resolved the issue. Waiting for another review.